hub/compute/ray.py

```diff
@@ -84,7 +89,6 @@ def store(
         ds: hub.Dataset
             uploaded dataset
         """
-
         _ds = ds or self._ds
```

Comment:
I think it should be _ds = ds or self.base_ds
Then we can skip the next line that stores the intermediate transform as done in transform.

Reply:
Do you mean to skip if conditioning as a whole after replacing `_ds = ds or self._ds`?

Reply:
Yes. We won't need to store the intermediate transform any more. Take a look at transform.py